### ISSUE TITLE
rds module with db_engine mysql5.5 and engine_version 5.5.41 returns misleading error message

### ISSUE THREAD

**Issue by danvaida** [ansible/ansible-modules-core#954]
##### Issue Type:

Bug Report
##### Ansible Version:

```
$ ansible --version
ansible 1.9 (devel b11be68249) last updated 2015/03/17 13:17:11 (GMT +200)
  lib/ansible/modules/core: (devel 8658b82de7) last updated 2015/03/17 13:49:28 (GMT +200)
  lib/ansible/modules/extras: (detached HEAD 696bc60caa) last updated 2015/03/17 13:17:56 (GMT +200)
  v2/ansible/modules/core: (detached HEAD 34784b7a61) last updated 2015/02/16 23:29:24 (GMT +200)
  v2/ansible/modules/extras: (detached HEAD 46e316a20a) last updated 2015/03/17 13:17:56 (GMT +200)
  configured module search path = None
```

Boto:

```
>>> boto.Version
'2.36.0'
```
##### Environment:

OS X 10.10.2
##### Summary:

For supporting a legacy app, I am trying to launch an RDS instance running MySQL 5.5.41. The task fails complaining about the parameter vpc_security_groups instead.
##### Steps To Reproduce:

```
- name: adding the BI RDS instance
      local_action:
        module: rds
        backup_retention: 14
        backup_window: "{{ db_backup_window }}"
        command: create
        license_model: general-public-license
        db_engine: MySQL
        engine_version: 5.5.41
        port: 3306
        instance_name: whatever
        instance_type: db.m3.medium
        parameter_group: blabla
        maint_window: Sat:02:00-Sat:03:00
        size: 100
        iops: 1000
        subnet: something
        vpc_security_groups: sg-XXXXXXXX
        username: me
        password: you
        region: eu-central-1
        wait: yes
        wait_timeout: 1800
        multi_zone: yes
```
##### Expected Results:

The RDS DB instance is created according to the task.
If I switch to engine_version 5.6.22 & db_engine mysql5.6 it works nicely.
##### Actual Results:

```
TASK: [adding the BI RDS instance] ********************************************
failed: [localhost -> 127.0.0.1] => {"failed": true, "parsed": false}
Traceback (most recent call last):
  File "/Users/danvaida/.ansible/tmp/ansible-tmp-1426597988.8-23404385027636/rds", line 2807, in <module>
    main()
  File "/Users/danvaida/.ansible/tmp/ansible-tmp-1426597988.8-23404385027636/rds", line 1008, in main
    invocations[module.params.get('command')](module, conn)
  File "/Users/danvaida/.ansible/tmp/ansible-tmp-1426597988.8-23404385027636/rds", line 652, in create_db_instance
    module.params.get('username'), module.params.get('password'), **params)
  File "/Users/danvaida/.ansible/tmp/ansible-tmp-1426597988.8-23404385027636/rds", line 429, in create_db_instance
    db_engine, username, password, **params)['CreateDBInstanceResponse']['CreateDBInstanceResult']['DBInstance']
TypeError: create_db_instance() got an unexpected keyword argument 'vpc_security_groups'


FATAL: all hosts have already failed -- aborting
```

**Comment by danvaida** [comment]
This error also seems to catch the issue with instance_type: 5.5 doesn't support db.t2.medium for example.

**Comment by jder** [comment]
I ran into this same error with ansible 1.9 and boto 2.36 as well; the parameter `vpc_security_groups` seems to have changed to `vpc_security_group_ids` in [RDS2](http://boto.readthedocs.org/en/latest/ref/rds2.html#boto.rds2.layer1.RDSConnection.create_db_instance).

**Comment by jder** [comment]
@danvaida You might be able to use the PR above to solve your problem. I'd be interested to know if it works or not.

**Comment by danvaida** [comment]
Just tested this PR on 1.9 b11be68249 and it works nicely. Thanks.

**Comment by leevs** [comment]
I manually changed this (rds.py) and it works fine now.

**Comment by bcoca** [comment]
closing this then as per merged PR #1032
